src/types/sha.ts

```diff
@@ -1,24 +1,23 @@
 // We are using the P-H-C storing format:
 // https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md
-import * as _crypto from 'crypto';
-import * as _shajs from 'sha.js';
+import type * as Crypto from 'crypto';
+import type * as Shajs from 'sha.js';
```

Comment:
These also look problematic to me, like above.

Reply:
Again, these were type-only imports before and are just updated for clarity